brightray/browser/browser_main_parts_mac.mm

```diff
@@ -6,6 +6,18 @@
 
 namespace brightray {
 
+void OverrideMacAppLogsPath() {
+  base::FilePath path;
+  NSString* bundleName = [[[NSBundle mainBundle] infoDictionary]
+    objectForKey:@"CFBundleName"];
+  NSString* logsPath = [NSString stringWithFormat:@"Library/Logs/%@",bundleName];
+
+  NSString* libraryPath = [NSHomeDirectory() stringByAppendingPathComponent:logsPath];
+  std::string libPathString = std::string([libraryPath UTF8String]);
```

Comment:
`base::FilePath` accepts `StringPiece` as input, so the `std::string` is not needed:

```c++
base::FilePath([libraryPath UTF8String])
```